docs/getting-started.md

```diff
@@ -56,8 +56,95 @@ documentation site.
 
 ## Alternative package managers (experimental)
 
-Although the recommended setup for Melange projects is with opam, Melange can
-also be used with other package managers, such as [esy](https://esy.sh/) and
-[Nix](https://github.com/NixOS/nix). There is also a [Melange project template
-for esy](https://github.com/melange-re/melange-esy-template) available, similar
-in spirit to the Melange opam template mentioned above.
+Melange can also be used with other package managers. The following instructions
+apply to:
+
+- [Nix](#nix)
+- [Esy](#esy)
+    - a [Melange project template for
+      esy](https://github.com/melange-re/melange-esy-template) is also
+      available to [start from a
+      template](https://github.com/melange-re/melange-esy-template/generate)
```

Comment:
I would just write a small sentence, the reference to the esy template can be moved to the new esy section. e.g.

```suggestion
Melange can also be used with other package managers. The following instructions
apply to [Nix](#nix) and [esy](#esy).
```

(I use lower case for esy to follow the style in [the official site](https://esy.sh/docs/en/what-why.html))